src/python/genny/genny_auto_tasks.py

```diff
@@ -22,19 +23,53 @@ def to_snake_case(str):
     s1 = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', str)
     return re.sub('([a-z0-9])([A-Z])', r'\1_\2', s1).lower()
 
+def cd_python_dir():
```

Comment:
Per offline discussion. You can go to the repo root instead to avoid having to encode an implicit relative location between this file and the python root; this file could get moved around.

Reply:
Also, you can just change to the repo root once at the start of this script. the current working directory that you specify is only going to apply to this file, not its caller.